src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DacKeyFieldDeclaration/KeyFieldDacTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.DacExtensionDefaultAttribute;
+using Acuminator.Analyzers.StaticAnalysis.DacKeyFieldDeclaration;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.DacKeyFieldDeclaration
+{
+	public class KeyFieldInDacTests : CodeFixVerifier
+	{
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new KeyFieldDeclarationAnalyzer();
+
+		protected override CodeFixProvider GetCSharpCodeFixProvider() => new KeyFieldDeclarationFix();
+
+		[Theory]
+		[EmbeddedFileData("CompoundKey.cs")]
+		public virtual void TestCompoundKey(string source) =>
```

Comment:
Use `async` version of the tests. Don't forget to use `async` / `await`.
See our Best Practices guide.

Reply:
And what is the point of marking all the methods as `virtual`?